Makefile

```
@@ -1,7 +1,7 @@
 SHELL := /bin/bash
 export GO111MODULE ?= on
 export VERSION ?= v1.1.0
-export ECE_VERSION ?= 2.7.0-ms47
+export ECE_VERSION ?= 2.8.0
```

This should be `2.8.0-ms48` since the actual ECE release will track MS49.
```suggestion
export ECE_VERSION ?= 2.8.0-ms48
```